src/index.js

```diff
@@ -1,17 +1,16 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
 import './index.css';
-import App from './App/components/App';
-import reportWebVitals from './reportWebVitals';
+import App from './App';
+import {Subtitle} from './components/Subtitle';
+import {Login} from './components/Login';
 
 ReactDOM.render(
   <React.StrictMode>
     <App />
```

Comment:
in this file, `App` should only be the one found here that renders UI.

Then in the `App` component, this is where we handle which view component to render. Like `LoginPage` `SnippetsDashboardPage` etc. 

In the current setup
```jsx
<App />
<Subtitle />
<Login />
```
they should be in one page called `LoginPage`. put it in `src/pages/LoginPage`